src/main/java/com/endava/utility/Main.java

```diff
@@ -0,0 +1,24 @@
+package com.endava.utility;
+
+import com.endava.utility.list.Lists;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        List<String> listOfStrings = Lists.createList("jedan","dva","tri");
```

Comment:
For guava list you used newArrayList(...) method to do this.
Java API offers this this functionality as well, you don't need to make your own method that creates a List from array of elements.
Take a look at https://docs.oracle.com/javase/7/docs/api/java/util/Arrays.html#asList(T...)

Reply:
Tnx! I was just playing around with generics and varargs and created custom implementation for practice. It is indeed superficial, and I removed it :)